app/src/main/java/com/github/sdpteam15/polyevents/fragments/ListFragment.kt

```diff
@@ -1,34 +1,76 @@
 package com.github.sdpteam15.polyevents.fragments
 
+import android.content.Intent
+import android.os.Build
 import android.os.Bundle
-import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.annotation.RequiresApi
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.RecyclerView
+import com.github.sdpteam15.polyevents.ActivityActivity
 import com.github.sdpteam15.polyevents.R
+import com.github.sdpteam15.polyevents.activity.Activity
+import com.github.sdpteam15.polyevents.activity.ActivityItemAdapter
+import com.github.sdpteam15.polyevents.helper.ActivitiesQueryHelper
+import com.github.sdpteam15.polyevents.helper.ActivitiesQueryHelperInterface
+import com.github.sdpteam15.polyevents.helper.HelperFunctions
 
-// TODO: Rename parameter arguments, choose names that match
-// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
-private const val ARG_PARAM1 = "param1"
-private const val ARG_PARAM2 = "param2"
+
+const val EXTRA_ACTIVITY = "com.github.sdpteam15.polyevents.activity.ACTIVITY_ID"
```

Comment:
A comment is missing to explain this constant.

Reply:
Thank you, I'll change it in the next commit